soda/core/soda/sodacl/tables_cfg.py

```diff
@@ -3,7 +3,7 @@
 from soda.sodacl.location import Location
 
 
-class DiscoverTablesCfg:
+class TablesCfg:
```

Comment:
We need to find a better name for this.

The generic version of this, provided we intend both discover tables and send samples to leverage this class should be something like:

`DatasourceTablesCfg` or something like that